filebeat/docs/modules-getting-started.asciidoc

```diff
@@ -119,15 +114,26 @@ logs:
 See the <<modules-tutorial>> for more information about setting variables and
 advanced options.
 
+[[passing-credentials-modules]]
+==== Pass credentials
+
+:start-type: modules
+include::../../libbeat/docs/shared-passing-credentials-setup.asciidoc[]
+
 [[visualizing-data]]
-==== Visualizing the data in Kibana
+==== Visualize the data in Kibana
 
 After you've confirmed that Filebeat is sending events to Elasticsearch, launch
 the Kibana web interface by pointing your browser to port 5601. For example,
 http://127.0.0.1:5601[http://127.0.0.1:5601].
 
 Open the dashboard and explore the visualizations for your parsed logs.
 
+TIP: If you don't see data in Kibana, try changing the date range to a larger
+range. By default, Kibana shows the last 15 minutes.
+
 Here's an example of the syslog dashboard:
 
+//REVIEWERS: Do we have an updated screen capture of this dashboard? If not, I can update it myself, but it would be good if you could point me to a good example log file that will populate the visualization with interesting information.
+
```

Comment:
@dedemorton There are some log files [here](https://github.com/elastic/beats/tree/master/filebeat/module/system/syslog/test), that we use for testing syslog. You can use these if you want. For syslog, I think it's sufficient to use the logs from your laptop.